frontend/src/styles/sidebar.css

```diff
@@ -1,7 +1,13 @@
+html {
+  font-family: -apple-system, system-ui, BlinkMacSystemFont, "Segoe UI", Roboto,
+    "Helvetica Neue", Arial, sans-serif;
+}
+
 .sidebar-logo-text {
-  width: 70%;
-  padding-top: 20px;
-  padding-bottom: 20px;
+  width: 162px;
```

Comment:
do we want this to be more responsive? direct `px` sizes make me a little nervous for how it'll look on different sized screens

Reply:
I think using direct `px` values should be fine since the sidebar width will be fixed at 375px and the site will not be viewed on a mobile device anyway. (Sidebar width will not be changed dynamically based on the viewport width.) `px` values should work well on tablet devices also, so I think we should be fine.